src/main/java/org/fundacionjala/at15/katas/langtonant/adhemar/Ant.java

```diff
@@ -0,0 +1,76 @@
+package org.fundacionjala.at15.katas.langtonant.adhemar;
+
+public class Ant {
+    int row;
+    int column;
+    int direction = 1; //Up: 1, rigth: 2, down: 3, left: 4
+
+    public Ant(int row, int column) {
+        this.row = row;
+        this.column = column;
+    }
+
+    public void start(int steps, Grid grid) {
+        for (int i = 0; i < steps; i++) {
+            boolean currentColor = grid.getCurrentColor(this.row, this.column);
+
+            grid.changeColor(this.row, this.column, !currentColor);
+
+            this.turn(currentColor);
+            this.move(grid);
+        }
+    }
+
+    public void turn(boolean color) {
+        if (color) {
+            if( this.direction - 1 == 0) {
+                this.direction = 4;
+            } else {
+                this.direction = this.direction - 1;
+            }
+        } else {
+            if( this.direction + 1 == 5) {
+                this.direction = 1;
+            } else {
+                this.direction = this.direction + 1;
+            }
+        }
+        // System.out.println("direction: " + this.direction);
```

Comment:
You can remove comments

Reply:
Ok, I'll do it, I used it to test my program but now it's unnecessary